packages/components/src/reaction_selection_menu/hooks.jsx

```diff
@@ -21,36 +21,34 @@ export const useLongTouch = ({ onStart, onEnd, onMove }) => {
   const active = useRef(false);
   const isMounted = useMounted();
 
-  const handleTimer = useCallback((event) => {
+  const handleTimer = useCallback(() => {
```

Comment:
do we need to wrap all these handlers in useCallback? They're not passed into useEffect etc, perf difference would be negligible no?

Reply:
I think for `handleStart`, `handleMove`, `handleEnd` and `handleCancel` we should keep the `useCallback` wrapping. Other hooks like `useEventListener` listen to changes on the callbacks like `handleStart` to know if they need to reattach it. Returning a new function all the time would cause it to reattach on every render.

The timer handlers may not need `useCallback` wrapping - will check